mantidproject/mantid
# Cmake-format documentation

**Description of work.**
Add CMake-format documentation to explain how to run it. 

**To test:**

Build the `dev-docs-html ` . Open `ToolsOverview.html` in `build directory->dev-docs->html`. Read through the cmake-format section and check it makes sense and check there are no typos and spelling mistakes.

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
